dive_core/capture_event_info.h

```diff
@@ -24,6 +24,55 @@
 namespace Dive
 {
 
+//--------------------------------------------------------------------------------------------------
+// Helper functions
+namespace Util
+{
+enum class EventType : uint8_t
+{
+    kDraw,
+    kDispatch,
+    kBlit,
+    kColorSysMemToGmemResolve,
+    kColorGmemToSysMemResolve,
+    kColorGmemToSysMemResolveAndClear,
+    kColorClearGmem,
+    kDepthSysMemToGmemResolve,
+    kDepthGmemToSysMemResolve,
+    kDepthGmemToSysMemResolveAndClear,
+    kDepthClearGmem,
+    kSysmemToGmemResolve,
+    kWaitMemWrites,
+    kWaitForIdle,
+    kWaitForMe,
+    kEventWriteStart,
+    kEventWriteEnd,
+    kUnknown
+};
+
+bool IsEvent(const IMemoryManager &mem_manager,
```

Comment:
should `state_tracker` be `const EmulateStateTracker&` ? same for the `GetEventType` and `GetEventString` funcs